NewKernel_d/include/CGAL/NewKernel_d/Cartesian_LA_base.h

```diff
@@ -20,16 +20,19 @@
 
 #ifndef CGAL_KERNEL_D_CARTESIAN_LA_BASE_H
 #define CGAL_KERNEL_D_CARTESIAN_LA_BASE_H
-
+#  pragma warning(push)
+#  pragma warning(disable: 4309)
```

Comment:
There ones are not protected by `#if defined(BOOST_MSVC) && (BOOST_MSVC == 1900)`. Is that wanted?